docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -224,33 +270,28 @@ For more information about CodeQL packs, see :doc:`About CodeQL Packs <about-cod
 Running query suites
 ~~~~~~~~~~~~~~~~~~~~
 
-To run a query suite over a CodeQL database for a C/C++ codebase,
+To run a query suite on a CodeQL database for a C/C++ codebase,
 you could use the following command from the directory containing your database::
 
-   codeql database analyze <cpp-database> cpp-code-scanning.qls --format=sarifv2.1.0 --output=cpp-results.sarif
+   codeql database analyze <cpp-database> codeql/cpp-queries:codeql-suites/cpp-code-scanning.qls --format=sarifv2.1.0 --output=cpp-results.sarif --download
 
-The analysis generates a file in the v2.1.0 SARIF format that is supported by all versions of GitHub.
-This file can be uploaded to GitHub by executing ``codeql github upload-results`` or the code scanning API.
+This command downloads the ``codeql/cpp-queries`` CodeQL query pack, runs the analysis, and generates a file in the v2.1.0 SARIF format that is supported by all versions of GitHub. This file can be uploaded to GitHub by executing ``codeql github upload-results`` or the code scanning API.
 For more information, see `Analyzing a CodeQL database <https://docs.github.com/en/code-security/code-scanning/using-codeql-code-scanning-with-your-existing-ci-system/configuring-codeql-cli-in-your-ci-system#analyzing-a-codeql-database>`__
 or `Code scanning API <https://docs.github.com/en/rest/reference/code-scanning>`__ in the GitHub documentation.
 
 CodeQL query suites are ``.qls`` files that use directives to select queries to run
 based on certain metadata properties. The standard CodeQL packs have metadata that specify
 the location of the query suites used by code scanning, so the CodeQL CLI knows where to find these
 suite files automatically, and you don't have to specify the full path on the command line.
-For more information, see ":ref:`About CodeQL packs <standard-codeql-packs>`."
-
-The standard query suites are stored at the following paths in
-the CodeQL repository::
+For more information, see ":ref:`Creating CodeQL query suites <creating-codeql-query-suites>`."
 
-   ql/<language>/ql/src/codeql-suites/<language>-code-scanning.qls
+Query suites in the core CodeQL query packs are all stored in the ``codeql-suites`` directory of the pack. For example, the ``codeql/cpp-queries`` pack contains the following query suites:
 
-and at the following path in the CodeQL for Go repository::
+* ``cpp-code-scanning.qls`` - Standard Code Scanning queries for C++.
+* ``cpp-security-and-quality`` - Security-and-quality queries for C++.
+* ``cpp-security-extended`` - Security-extended queries for C++. This suite contains queries that are less precise than the standard security queries, and may find more false-positives.
```

Comment:
Reorder these: security-extended should go first, and highlight that each includes the suites above it.